flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescalingITCase.java

```diff
@@ -98,6 +100,9 @@
 		return new Object[]{"filesystem", "rocksdb"};
 	}
 
+	@Rule
+	public LogTestName logTestName = new LogTestName();
```

Comment:
shouldn't this be done by `TestLogger` already (but instead of logging to stdout, it will only print these in the log files)

Reply:
`TestLogger` was not logging into stdout/stderr, but into disabled logging, so it didn't have an effect. 

Nevertheless dropping this change.